lab_2_keywords_cooccurrence/main.py

```diff
@@ -29,7 +49,13 @@ def extract_candidate_keyword_phrases(phrases: Sequence[str], stop_words: Sequen
 
     In case of corrupt input arguments, None is returned
     """
-    pass
+    if not check_type_and_not_empty(phrases, list) or not check_type_and_not_empty(stop_words, list):
+        return None
+    modified_phrases = []
+    for sentence in phrases:
+        modified_sentence = ' '.join([word if word not in stop_words else '?' for word in sentence.lower().split()])
```

Comment:
This is a very long and difficult to read line.
1.  Save `sentence.lower().split()` to a variable
2.  Save the list with words to a variable
3. join the list